srcpkgs/dpdk/template

```diff
@@ -0,0 +1,87 @@
+# Template file for 'dpdk'
+pkgname=dpdk
+version=19.08
+revision=1
+archs="x86_64 i686"
+build_style=meta
+hostmakedepends="cmake meson doxygen python3-Sphinx pkg-config elfutils"
+makedepends="libnuma-devel linux-headers jansson-devel libpcap-devel zlib-devel libressl-devel libbsd-devel libelf libmnl"
+depends="jansson zlib libbsd libressl python-pyelftools"
+short_desc="Data Plane Development Kit"
+maintainer="Hans-J. Schmid <knock@myopendoor.de>"
+license="BSD-3-Clause, GPL-2.0-only"
+homepage="https://www.dpdk.org/"
+distfiles="https://github.com/DPDK/${pkgname}/archive/v${version}.tar.gz"
+checksum=1ceff1a6f4f8d5f6f62c1682097249227ac5225ccd9638e0af09f5411c681038
+
+do_configure() {
+	meson --prefix /usr -Denable_kmods=false -Denable_docs=true build
+}
+
+do_build() {
+	ninja -C build
+}
+
+do_install() {
+	DESTDIR=${DESTDIR} ninja -C build install
+	vmkdir usr/lib/modules-load.d
+}
+
+post_install() {
+	vlicense license/bsd-3-clause.txt
+	vlicense license/gpl-2.0.txt
+}
+
+dpdk-doc_package() {
+	short_desc+=" - documentation"
+	archs=noarch
+	pkg_install() {
+		vmove usr/share/doc
+	}
+}
+
+dpdk-devel_package() {
+	depends="${sourcepkg}>=${version}_${revision} jansson-devel zlib-devel libbsd-devel libressl-devel"
```

Comment:
It should depends on `dpdk-libs>=${version}_${revision}` not `${sourcepkg}>=${version}_${revision}`. But I'm not sure if you actually should make a `dpdk-libs` subpackage.

Reply:
Will remove the libs package. Wasn't sure about it.

Reply:
I was using Ubuntu's packages as a template on how to structure them. But in the main package there are only two python scripts left. Does not make a lot of sense.

Reply:
You should readd `${sourcepkg}>=${version}_${revision}`. If something wants to link against any of the libs it will just find `*.so` symlinks that won't resolve.